examples/fc_networks.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+	"strconv"
+)
+
+func main() {
+	var (
+		ClientOV    *ov.OVClient
+		testName    = "TestFCNetworkGOsdk"
+		new_fc_name = "RenamedFCNetwork"
+		falseVar    = false
+	)
+	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		apiversion)
+	initialScopeUris := new([]utils.Nstring)
+	*initialScopeUris = append(*initialScopeUris, utils.NewNstring("/rest/scopes/8a4e85fe-0725-482e-b232-4877b78fde18"))
+	fcNetwork := ov.FCNetwork{
+		AutoLoginRedistribution: falseVar,
+		Description:             "Test FC Network",
+		LinkStabilityTime:       30,
+		FabricType:              "FabricAttach",
+		Name:                    testName,
+		Type:                    "fc-networkV4",
+		InitialScopeUris:        *initialScopeUris,
+	}
+	fmt.Println(fcNetwork)
+	err := ovc.CreateFCNetwork(fcNetwork)
+	if err != nil {
+		fmt.Println("Fc Network Creation Failed: ", err)
+	} else {
+		fmt.Println("Fc Network created successfully...")
+	}
+	sort := "name:desc"
+
+	fcNetworks, err := ovc.GetFCNetworks("", sort, "", "")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("#---Get Fc Networks sorted by name in descending order----#")
+
+	for i := 0; i < len(fcNetworks.Members); i++ {
```

Comment:
Consider using a `for ... range` loop, since it won't require you to access items by index.

Reply:
done. Thanks